### PR TITLE
Improve the Export All button display in the export dialog

### LINKED ISSUE #33293: "Export All" shouldn't require selection
**Godot version:**
3.2 alpha3

**Issue description:**
When you try to export project, there's the "Export All" option that basically exports for all defined presets. However, it is only clickable when you have any preset selected, which makes sense for "Export Project", but not "Export All". You should be able to export everything if there's at least one preset, not if something is selected.

### PR DESCRIPTION
**Pull request by Calinou**
- Update the disabled status when changing an export path.
- Display a tooltip that states why the button is disabled.
- Update the Export All dialog message to have a more friendly tone.
- Suffix button texts with "...", as they display another dialog when clicked.

This closes #33293.

### CODE REVIEW

**Comment by KoBeWi** [comment]
Is .pck file platform-specific? If not, the "Export PCK/Zip" button could get similar changes.

**Comment by akien-mga** [comment]
> Is .pck file platform-specific?

Yes. For Mono exports, it will include the BCL for the target platform (differs on Windows, Linux+macOS, Wasm).

There can also be platform/preset specific options used that impact the pck (e.g. texture compression formats).

File: editor/project_export.cpp [hunk 1]
```diff
@@ -626,6 +630,10 @@ void ProjectExportDialog::_delete_preset_confirm() {
 	get_ok()->set_disabled(true);
 	EditorExport::get_singleton()->remove_export_preset(idx);
 	_update_presets();
+
+	// The Export All button might become enabled (if all other presets have an export path defined),
+	// or it could be disabled (if there are no presets anymore).
```
Comment (KoBeWi) [inline editor/project_export.cpp:635]: Not sure if this comment is really necessary.

**Review by KoBeWi — APPROVED** [review]
Needs rebase.

**Comment by Calinou** [comment]
Rebased and tested again, it works successfully.

File: editor/project_export.cpp [hunk 2]
```diff
@@ -1199,7 +1207,7 @@ ProjectExportDialog::ProjectExportDialog() {
 	updating = false;
 
 	get_cancel_button()->set_text(TTR("Close"));
-	get_ok_button()->set_text(TTR("Export PCK/Zip"));
+	get_ok_button()->set_text(TTR("Export PCK/ZIP..."));
 	export_button = add_button(TTR("Export Project"), !DisplayServer::get_singleton()->get_swap_cancel_ok(), "export");
```
Comment (akien-mga) [inline editor/project_export.cpp:1211]: If the other two get `...` added, shouldn't "Export Project" get it too? It also spawns a dialog.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**